react/AppIcon/Readme.md

```diff
@@ -9,5 +27,5 @@
   </div>
 ```
 
-The `fetchIcon` method is given `app.links.icon` as argument and must return an URL. This URL may be created with [`URL.createObjectURL`](https://developer.mozilla.org/en/docs/Web/API/URL/createObjectURL).
-`fetchIcon` may also throw errors.
+## `domain` and `secure` props
+If the `fetchIcon` is missing, the `<AppIcon />` component needs the `domain` prop (litteraly the cozy domain) and the secure prop, to know which protocol use between `http` or `https`.
```

Comment:
Since the AppIcon are going to be rendered in the context of a client, those props could be provided automatically.

```
const AppIcon = withClient(props => {
  return <DumbAppIcon domain={props.client.domain} secure={props.client.url.indexOf('https') === 0} />
})
```

or something like this ?

Reply:
Yep, it looks clever, I like it. (By the way I am still not a fan of always testing the protocol in client.url. IMHO the client should expose a `secure` attribute, but it's a subject for another time :))